components/builder-api/doc/api.raml

```diff
@@ -267,6 +267,21 @@ securitySchemes:
                 description: Project does not exist with corresponding projectId
             422:
                 description: Invalid or missing projectId in body
+    /group:
+        /{id}:
+            /promote:
+                /{channel}:
+                    post:
+                        description: Promote every successful project in a job group to the specified channel
+                        responses:
+                            200:
+                                description: Successful promotion
+                            400:
+                                description: ID or channel not provided
+                            404:
+                                description: Group not found
+                            500:
+                                description: Internal server error
```

Comment:
Based on the description, there are some projects (namely failed ones) in a group that would not get promoted. Can / should we give any indication of which these are? What if the group itself failed (e.g., some package toward the root of the dependency tree, and not a leaf, failed); what error would be returned then? What happens if you promote to a channel that the packages are already in?